src/build/tenscript/mod.rs

```diff
@@ -180,20 +197,20 @@ impl Library {
         let source = fs::read_to_string("src/build/tenscript/library.scm").unwrap();
         match Self::from_tenscript(&source) {
             Ok(library) => library,
-            Err(ParseError::Pest(error)) => panic!("pest parse error: \n{error}"),
-            Err(e) => panic!("{e:?}")
+            Err(TenscriptError::Pest(_error)) => panic!("pest parse error: \n{_error}"),
+            Err(_error) => panic!("{_error:?}")
```

Comment:
```suggestion
            Err(TenscriptError::Pest(error)) => panic!("pest parse error: \n{error}"),
            Err(error) => panic!("{error:?}")
```
Only prefix underscore if unused